google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/prepare/datacatalog_entry_factory.py

```diff
@@ -0,0 +1,39 @@
+from google.cloud import datacatalog_v1beta1
+from google.datacatalog_connectors.commons.prepare.base_entry_factory import \
+    BaseEntryFactory
+
+
+class DataCatalogEntryFactory(BaseEntryFactory):
+    NO_VALUE_SPECIFIED = 'UNDEFINED'
+    EMPTY_TOKEN = '?'
+
+    def __init__(self, project_id, location_id, metadata_host_server,
+                 entry_group_id):
+        self.__project_id = project_id
+        self.__location_id = location_id
+        self.__metadata_host_server = metadata_host_server
+        self.__entry_group_id = entry_group_id
+
+    def make_entry_for_topic(self, topic):
+        """Create Datacatalog entries from a table container dict.
```

Comment:
Yes, I have changed it in the [next PR, here, line 20](https://github.com/bonifacyj/datacatalog-connectors-message-brokers/pull/2/files#diff-f19b700b22cd8311fdc5d3518c321bda), so it will be overwritten once we merge the second PR